Kinder/Registration.xaml.cs

```diff
@@ -30,40 +25,120 @@ public RegistrationWindow()
             InitializeComponent();
         }
 
-        private void SignUpButton_Click(object sender, RoutedEventArgs e)               ///To add normal validation for email, phonenumber, username...
+        private void SignUpButton_Click(object sender, RoutedEventArgs e)               ///NEED ADVICE! TO DO - Maybe put all those blocks somewhere else, or try to do a multifunctial one 
```

Comment:
put it somewhere, so it could be usable, clean UI is not our focus right now

Reply:
I guess i'll leave it for now, we dont have so much time for optimization now :)